src/index.ts

```diff
@@ -247,9 +266,12 @@ export class Certbot extends Construct {
         REISSUE_DAYS: (props.reIssueDays === undefined) ? '30' : String(props.reIssueDays),
         PREFERRED_CHAIN: props.preferredChain || 'None',
         NOTIFICATION_SNS_ARN: snsTopic.topicArn,
+        DRY_RUN: 'False',
       },
       layers,
       timeout: props.timeout || Duration.seconds(180),
+      filesystem: props.efsAccessPoint ? lambda.FileSystem.fromEfsAccessPoint(props.efsAccessPoint, '/mnt/efs') : undefined,
+      vpc: props.vpc,
```

Comment:
It's effectively optional, because the lambda.Function's `vpc` property is defined as `ec2.IVpc | undefined`. So if a VPC was provided, the Function has it set, and if one wasn't provided, the Function gets `vpc: undefined`, which is what we want. If you check the snapshot, you'll see that the function's `VpcConfig` is not present, except when I've actually given a VPC.

Reply:
Okay cool. That works then